src/main/xerus/monstercat/tabs/TabSettings.kt

```diff
@@ -83,6 +85,13 @@ class TabSettings: VTab() {
 			Settings.PLAYERSEEKBARHEIGHT.bind(valueProperty() as ObservableValue<out Double>)
 		})
 		
+		add(CheckBox("Enable Streamer Mode (skips copyright-unsafe tracks in the player)").bind(Settings.SKIPUNLICENSABLE)).apply {
```

Comment:
Does the word "copyright-unsafe" even exist :joy: ? Why not stay with "unlicensable"?

Reply:
c2932419ee89f05eec201e2c27040c6974394f4f